spec/README.md

```diff
@@ -395,6 +389,13 @@ Contains a URL encoded value (e.g. `%C3%B6`) - see [percent encoding](https://en
 
 #### 3.1.2 Other Automatically Generated Data
 
+##### data.{character}.capital
+
+Optional  
+Type: Boolean
+
+This `true` or `false` value signifies that the character is capitalized. If this value is included in the source file, the source value will be used in place of the generated value.
```

Comment:
So you mean that this property should only be created when the generated value isn't correct? This is error prone in my opinion, since there may be changes in the generation process in future. Also, since users also need to create either `lowerCase` or e.g. `upperCase` they already know if it's capital or not. Don't see a big benefit by making this optional.

Reply:
We can just make the build code ignore the user setting and generate it automatically.

Reply:
how reliable is that?

Reply:
Ping @Mottie

Reply:
I don't know. I have no empirical data on how reliable it is to use the browser's built-in `toLowerCase()` function result and compare it to the original character. I don't know if this comparison would work on Arabic or Thai characters.

So, you just want to make this required and remove the logic?

Reply:
Since we're saying our database may not be complete, but is reliable, I don't see a different way. However, we can maybe generate the property automatically. The only thing I'm worried about is that this will loose comments in the .json files.